ui/core/src/ui-core.ts

```diff
@@ -78,6 +78,7 @@ export * from "./ui-core/inputs/LabeledInput";
 export * from "./ui-core/inputs/LabeledTextarea";
 export * from "./ui-core/inputs/numericinput/NumericInput";
 export * from "./ui-core/inputs/numericinput/ReactNumericInput";
+export * from "./ui-core/inputs/numericinput/NumberInput";
```

Comment:
Should we create a new subdirectory named "numberinput"?

Reply:
done